ftplugin/lean/lean.lua

```diff
@@ -3,7 +3,7 @@ vim.b.did_ftplugin = 1
 
 vim.opt.wildignore:append[[*.olean]]
 
-vim.opt_local.iskeyword = [[@,48-57,_,-,!,#,$,%]]
+vim.opt_local.iskeyword = [[@,48-57,_,-,.,!,#,$,%]]
```

Comment:
This would have been my next question, thanks for fixing this!

Reply:
This is already annoying me, so I'm going to PR using `syn iskeyword`.